.travis.yml

```diff
@@ -1,22 +1,23 @@
 language: c
 compiler: gcc
-dist: trusty
+dist: bionic
 os: linux
 notifications:
     email: false
 
 before_install:
     # add LLVM repository
-    - wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
-    - echo "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-7 main" | sudo tee -a /etc/apt/sources.list
+    #- wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+    #- echo "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-7 main" | sudo tee -a /etc/apt/sources.list
```

Comment:
Shouldn't this be deleted instead of just being commented out?

Reply:
I could have but I figured I'd leave an example of how to do it in case someone needed to re-add it in the future. Definitely not needed.

Reply:
There are plenty of tutorials on the Internet that teach how to fetch a gpg key and add a repository to your Linux distro, I wouldn't pollute the code with this. So, I still think you should just remove it.

Reply:
git has a history (and a blame or diff view quickly reveals why this apt code was removed). This github PR also highlights the old code and the discussion about it.

Keeping dead code is not helpful in my opinion (it's just confusing).

Reply:
Fair enough. I've deleted it.